pkg/mtd/types.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mtd
+
+// Info provides information about an active MTD.
+type Info interface {
```

Comment:
i don't really understand what this does from the comment or its usage

Reply:
it's gone.